### PR TITLE
Fabo/e2e browserstack

### PR DESCRIPTION
**Pull request by faboweb**
Same E2e test pr but with browserstack

### CODE REVIEW

**Comment by faboweb** [comment]
See videos here https://automate.browserstack.com/dashboard/v2/builds/ad5d8f31da4ef62b9d73b4a2aacd5003d4321361

File: .circleci/config.yml [hunk 1]
```diff
@@ -113,56 +113,16 @@ jobs:
...
-            docker-compose up --build -d
-      - run:
-          name: Test
-          command: npm run serve:dist & npm run test:e2e:local
-          no_output_timeout: 120
-      - store_artifacts:
-          path: ./screenshots
-          when: on_fail
+            PR_NUMBER="$(echo $CIRCLE_PULL_REQUEST | cut -d'/' -f 7)"
+            PASSWORD=${NETWORK_PASSWORD} CHT_PWD=${ACCOUNT_PASSWORD} yarn test:e2e:browserstack --fe=https://deploy-preview-${PR_NUMBER}--lunieio.netlify.com --api=https://staging-api.lunie.io
```
Comment (jbibla) [inline .circleci/config.yml]: shouldn't this be checking against production API?

Comment (faboweb) [reply .circleci/config.yml]: ture, one last thing I overlooked

Comment (faboweb) [inline .circleci/config.yml]:
```suggestion
            PASSWORD=${NETWORK_PASSWORD} CHT_PWD=${ACCOUNT_PASSWORD} yarn test:e2e:browserstack --fe=https://deploy-preview-${PR_NUMBER}--lunieio.netlify.com --api=https://production-api.lunie.io
```

**Pull request merged**